### PR TITLE
Remove various repeated words in documentation

### PR DESCRIPTION
**Pull request by eumiro**
- [ ] Closes #xxxx
 - [x] I am familiar with the [contributing guidelines](https://pvlib-python.readthedocs.io/en/latest/contributing.html)
 - [ ] Tests added
 - [ ] Updates entries in [`docs/sphinx/source/reference`](https://github.com/pvlib/pvlib-python/blob/main/docs/sphinx/source/reference) for API changes.
 - [ ] Adds description and name entries in the appropriate "what's new" file in [`docs/sphinx/source/whatsnew`](https://github.com/pvlib/pvlib-python/tree/main/docs/sphinx/source/whatsnew) for all changes. Includes link to the GitHub Issue with `` :issue:`num` `` or this Pull Request with `` :pull:`num` ``. Includes contributor name and/or GitHub username (link with `` :ghuser:`user` ``).
 - [x] New code is fully documented. Includes [numpydoc](https://numpydoc.readthedocs.io/en/latest/format.html) compliant docstrings, examples, and comments where necessary.
 - [x] Pull request is nearly complete and ready for detailed review.
 - [x] Maintainer: Appropriate GitHub Labels (including `remote-data`) and Milestone are assigned to the Pull Request and linked Issue.

Just removed repeated words from comments and docs.

### CODE REVIEW

**Comment by cwhanse** [comment]
Thanks @eumiro. 

If you'd like, you can mark out irrelevant items in the PR template by putting "~~" in front and after the text to be lined out.

**Review by AdamRJensen — APPROVED** [review]
Looks good!

I guess you ran some kind of script to identify repeated words?

**Comment by eumiro** [comment]
> Looks good!
> 
> I guess you ran some kind of script to identify repeated words?

Yes, something like ```\b(\w+)\s+\1\b` . Double words are are often invisible in longer texts.

File: pvlib/ivtools/sdm.py [hunk 1]
```diff
@@ -48,7 +48,7 @@ def fit_cec_sam(celltype, v_mp, i_mp, v_oc, i_sc, alpha_sc, beta_voc,
     beta_voc : float
         Temperature coefficient of open circuit voltage [V/C]
     gamma_pmp : float
-        Temperature coefficient of power at maximum point point [%/C]
+        Temperature coefficient of power at maximum point [%/C]
```
Comment (kandersolar) [inline pvlib/ivtools/sdm.py:51]:
```suggestion
        Temperature coefficient of power at maximum power point [%/C]
```

I think this one was an incorrect word, not an extra word

**Comment by kandersolar** [comment]
Thanks @eumiro!

**Pull request merged**